src/lib/jsonParse/jsonRequest.cpp

```diff
@@ -184,7 +184,9 @@ std::string jsonTreat
   //
   // If the payload is empty, the XML parsing library does an assert
   // and the broker dies. I don't know about the JSON library, but just in case ...
-  // 
+  //
+  // FIXME P4 #1862: check if we really need this now that Orion only support JSON
```

Comment:
This has nothing to do with XML.
If we want to remove the check, we'd have to test it.
What does the old json parser do if it is sent an empty payload (or NULL) ?
If it doesn't crash (as the XML parser), and the broker works just likr without the check, then it's safe to remove the check